main/boost/APKBUILD

```diff
@@ -153,28 +161,42 @@ package() {
 }
 
 static() {
-	pkgdesc="Static libraries for boost"
+	pkgdesc="Boost static libraries"
+	depends="$depends_static"
+
 	mkdir -p "$subpkgdir"/usr/lib
-	mv "$pkgdir"/usr/lib/*.a "$subpkgdir"/usr/lib/
+	mv "$pkgdir"/usr/lib/lib*.a "$subpkgdir"/usr/lib/
```

Comment:
are there any `*.a` files that does not start with `lib`? If so, what are those for and why should they be excluded from the `boost-static`?

Reply:
There aren't any. I really don't think there will be any installed into `/usr/lib` but I like being more specific here with the files we are trying to move.